src/modules/QtQuick/RegExpValidator.js

```diff
@@ -3,11 +3,13 @@ registerQmlType({
   name:     'RegExpValidator',
   versions: /.*/,
   baseClass: 'Item',
-  constructor: function QMLRegExpValidator(meta) {
+  properties: {
+    regExp: 'var'
+  }
+}, class {
+  constructor(meta) {
     callSuper(this, meta);
 
-    createProperty("var", this, "regExp");
-
     this.validate = (function(string) {
```

Comment:
Yes, but that would require more code movement, and I wanted to do that in a follow-up PR.